ddebug/debug.h

```diff
@@ -134,11 +134,11 @@
 #endif
 // #define DEBUG_LEVEL 0		//DAJ
 #if defined(WIN32) || defined(__LINUX__)
-static const int OSMBOX_OK = 1;
-static const int OSMBOX_YESNO = 2;
-static const int OSMBOX_YESNOCANCEL = 3;
-static const int OSMBOX_ABORTRETRYIGNORE = 4;
-static const int OSMBOX_OKCANCEL = 5;
+constexpr const int OSMBOX_OK = 1;
+constexpr const int OSMBOX_YESNO = 2;
+constexpr const int OSMBOX_YESNOCANCEL = 3;
+constexpr const int OSMBOX_ABORTRETRYIGNORE = 4;
+constexpr const int OSMBOX_OKCANCEL = 5;
 #else
 #define OSMBOX_OK 1
```

Comment:
should these definitions go away?

Reply:
That is not in the scope of this PR.

Reply:
I know, it was more of a general remark and an idea for a future cleanup

Reply:
Oh, I see, the definition below. Yeah, they should go and the `constexpr` ones should be universal. Also there seems to be a duplicate sets that merely leave of "OS" from the beginning. I'll clean it up. Need to merge this first, though.